### DISCUSSION TITLE
1.x升级至2.x版本，vue兼容问题

### DISCUSSION THREAD

**Discussion by Dame0912** [antvis/X6#3875] [Q&A]
1.x 版本中vue组件使用
Graph.registerVueComponent('start-node', {
        template: `
            <start-node></start-node>`,
        components: {
          StartNode
        }
      }, true)
     
graph.createNode({
            shape: 'vue-shape',
            component: 'virtual-node',
            width: 180,
            height: 36,
            ports: ports
          })

这样是不是就没法升级到 2.x版本了。2.x注册VUE组件的 shape是自定义的，1.x中我的shape都是 'vue-shape'

**Comment by NewByVector** [comment]
2.x 中 shape 就像你给节点形状自定义的名称。
